### PR TITLE
v0.38.2

### PR DESCRIPTION
**Pull request by Archmonger**
- Fix `idom.run` failing to run if not specifying a server implementation.
- Bump to `0.38.2`
- Disable greetings CI
- Clean up references to idom.server
- Fix one miscellaneous use of `logger.warn` (deprecated)
- Fix typos in nox descriptions

### CODE REVIEW

**Review by rmorshea — DISMISSED** [review]

**Comment by rmorshea** [comment]
I'm not really sure what's going wrong, but `tests/test_web/test_module.py::test_module_from_url` is failing when I run the tests locally.

**Comment by rmorshea** [comment]
There is [a script](https://idom-docs.herokuapp.com/docs/about/contributor-guide.html#release-process), but it bumps versions and pushes a tag at the same time. I guess I'd assumed that would be done by whoever is performing the release.

**Comment by Archmonger** [comment]
@rmorshea Everything remaining seems to just be variable renaming... Not entirely sure how tests are failing.

File: src/idom/testing/display.py [hunk 1]
```diff
@@ -58,8 +58,8 @@ async def __aenter__(self) -> DisplayFixture:
             self.page = await browser.new_page()
 
         if not hasattr(self, "server"):
```
Comment (rmorshea) [inline src/idom/testing/display.py:60]: Found the problem!

```suggestion
        if not hasattr(self, "backend"):
```

**Comment by rmorshea** [comment]
BTW, I already merged a change that removes the greeting message in favor of a small PR template for now. Can revisit the greeting message later.

**Comment by rmorshea** [comment]
I think I've got the fix here + some other minor docstring changes: https://github.com/idom-team/idom/pull/763

Going to close this.

**Pull request closed without merging**